libraries/cyclestreets-view/src/main/java/org/mapsforge/android/maps/MapsforgeOSMDroidTileProvider.java

```diff
@@ -19,6 +19,8 @@
 import okhttp3.Request;
 import okhttp3.Response;
 
+import static org.osmdroid.tileprovider.constants.OpenStreetMapTileProviderConstants.*;
```

Comment:
Can I just get a safety check that this doesn't end up importing the default OSM tile location? We should be using our own tileserver URL. At present this is just proxying the OSM one but in future will have its own renderer. Our own one also supports requests with the 2x resolution flag in the URL.